### PR TITLE
Remove ypbind rules from RHEL10

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:
Remove ypbind rules from RHEL10

#### Rationale:
No longer in RHEL 10.

### CODE REVIEW

File: controls/cis_rhel10.yml [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ controls:
       - l1_workstation
     status: automated
     notes: Review the availability of this package when the product is out.
-    rules:
+    related_rules:
```
Comment (marcusburghardt) [inline controls/cis_rhel10.yml:1050]: Could you also remove the comment I put in the notes, please? : )

Comment (Mab879) [reply controls/cis_rhel10.yml:1049]: Done.

**Review by vojtapolasek — APPROVED** [review]
LGTM, thank you.

**Pull request merged**